README.md

```diff
@@ -39,13 +39,40 @@ You can then load the module in F# Interactive, and run it:
 ```
 > Property.print propReverse
 +++ OK, passed 100 tests.
-
 ```
 
 More examples can be found in the [tutorial](doc/tutorial.md).
 
 👉 For auto-generators (à la AutoFixture) and other convenience generators, check out [fsharp-hedgehog-experimental](https://github.com/cmeeren/fsharp-hedgehog-experimental/).
 
+## C#/LINQ Example
+
+Hedgehog provides an alternative namespace, meant to provide compatibility with other .NET languages. To use this, simply import the `Hedgehog.Linq` namespace. The base `Hedgehog` namespace isn't necessary in this scenario.
+
+Generators are then built with a more comfortable fluent API.
+
+```csharp
+using Hegehog.Linq;
+
+class Generators
+{
+    static void Example()
+    {
+        // This creates a generator for 10 character strings, with only alphabetical characters.
+        var stringLength = 10;
+        var stringGen = Gen.String(Gen.Alpha, Range.FromValue(stringLength));
```

Comment:
Now seeing this used, I don't think I like `Range.FromValue`, maybe `Range.Constant` would be better after all.

Reply:
I think `FromValue` is a friendlier name for developers that are new to functional programming. I've seen developers confusing singleton with the *singleton* lifetime when doing dependency injection, for example.

OTOH, if we don't push towards the correct terminology, the world will stay forever in this weird state of trying to map OO to FP in a funky way. (Perhaps that's one of the reasons Mark Seemann uses the terms as-is, e.g. *Either* instead of *ROP*.)

Reply:
We can tackle this separately, of course.